### PR TITLE
Merge `Code*` type stub back to source

### PR DESCRIPTION
**Pull request by trim21**

### CODE REVIEW

File: github/CodeScanAlertInstance.py [hunk 1]
```diff
@@ -115,7 +101,7 @@ def _useAttributes(self, attributes: Dict[str, Any]) -> None:
         if "commit_sha" in attributes:  # pragma no branch
             self._commit_sha = self._makeStringAttribute(attributes["commit_sha"])
         if "message" in attributes:  # pragma no branch
-            self._message = self._makeDictAttribute(attributes["message"])
+            self._message = self._makeDictAttribute(attributes["message"])  # type: ignore
```
Comment (trim21) [inline github/CodeScanAlertInstance.py:104]: `Dict[str, Any]` can't cast to `TypedDict`

Comment (trim21) [reply github/CodeScanAlertInstance.py:104]: Or we can just use `dict[str,Any]`, remove `Message(TypedDict)`

**Comment by trim21** [comment]
@EnricoMi This PR is ready for review.

File: github/CodeScanAlert.py [hunk 2]
```diff
@@ -19,140 +19,103 @@
...
+        self._tool: Attribute[Any] = NotSet
+        self._created_at: Attribute[Any] = NotSet
+        self._dismissed_at: Attribute[Any] = NotSet
+        self._dismissed_by: Attribute[Any] = NotSet
+        self._dismissed_reason: Attribute[str] = NotSet
+        self._url: Attribute[str] = NotSet
+        self._html_url: Attribute[str] = NotSet
+        self._instances_url: Attribute[str] = NotSet
+        self._most_recent_instance: Attribute[Any] = NotSet
+        self._state: Attribute[str] = NotSet
```
Comment (EnricoMi) [inline github/CodeScanAlert.py:54]: Please specify types of these `Attribute[Any]`.

File: github/CodeScanAlertInstance.py (hunk 1 above)
Comment (EnricoMi) [reply github/CodeScanAlertInstance.py:104]: Happy to remove the `Message(TypedDict)`.

**Review by EnricoMi — CHANGES_REQUESTED** [review]

File: github/CodeScanAlert.py [hunk 3]
```diff
@@ -19,140 +19,105 @@
...
     This class represents alerts from code scanning.
     The reference can be found here https://docs.github.com/en/rest/reference/code-scanning.
     """
 
+    def _initAttributes(self) -> None:
+        self._number: Attribute[int] = NotSet
+        self._rule: Attribute[github.CodeScanRule.CodeScanRule] = NotSet
+        self._tool: Attribute[
+            github.CodeScanTool.CodeScanTool,
+        ] = NotSet
```
Comment (EnricoMi) [inline github/CodeScanAlert.py:47]: Are these line breaks really needed?

**Review by EnricoMi — APPROVED** [review]
LGTM! Minor code style question.

File: github/CodeScanAlert.py (hunk 3 above)
Comment (trim21) [reply github/CodeScanAlert.py:47]: this formatted by black, not me

Comment (EnricoMi) [reply github/CodeScanAlert.py:47]: Question is why, line 55 is much longer and not split.

Comment (trim21) [reply github/CodeScanAlert.py:47]: comma, I guess

**Pull request merged**